backend/lambda/database_calls.py

```diff
@@ -3,13 +3,20 @@
 import json
 import os
 import sqlalchemy
+from google.cloud import secretmanager
 
 class DatabaseCalls():
 
     def get_secret(self):
         print('grabbing secret')
+        
+        client = secretmanager.SecretManagerServiceClient()
```

Comment:
you're missing quite a bit of semicolons here

Reply:
Hm, this is python. It doesn't need semicolons